layer/VkLayer_khronos_profiles.json.in

```diff
@@ -105,42 +106,51 @@
                     "filter": "*.json",
                     "default": "${VULKAN_CONTENT}/VK_LAYER_KHRONOS_profiles/VP_LUNARG_desktop_portability_2021.json",
                     "expanded": true,
+                    "platforms": [ "WINDOWS", "LINUX", "MACOS", "ANDROID" ],
                     "settings": [
                         {
                             "key": "profile_name",
+                            "env": "VK_KHRONOS_PROFILES_PROFILE_NAME",
                             "label": "Profile",
                             "description": "Name of the profile specified by the profile file to use. If set to ${VP_DEFAULT}, the layer will load the first profile in the file.",
                             "type": "ENUM",
+                            "platforms": [ "WINDOWS", "LINUX", "MACOS", "ANDROID" ],
                             "flags": [],
                             "default": "${VP_DEFAULT}"
                         },
                         {
                             "key": "profile_validation",
+                            "env": "VK_KHRONOS_PROFILES_PROFILE_VALIDATION",
                             "label": "Schema Validation",
                             "description": "Validate the profile file against the Vulkan SDK profile schema if the file is found.",
                             "type": "BOOL",
-                            "default": false
+                            "default": false,
+                            "platforms": [ "WINDOWS", "LINUX", "MACOS" ]
                         }
                     ]
                 },
                 {
                     "key": "emulate_portability",
+                    "env": "VK_KHRONOS_PROFILES_EMULATE_PORTABILITY",
                     "label": "Emulate VK_KHR_portability_subset",
                     "description": "Emulate the VK_KHR_portability_subset extension on the device.",
                     "status": "STABLE",
                     "platforms": [
                         "WINDOWS",
-                        "LINUX"
+                        "LINUX",
+                        "ANDROID"
```

Comment:
The version of this file on master does not include macos in the platforms for this option, so I didn't port it over.
In fact, since I made my changes, Android has been completely eliminated from this file, so I'll be removing it.